providers/fips/fipsprov.c

```diff
@@ -373,8 +373,8 @@ static const OSSL_ALGORITHM fips_keyexch[] = {
 #endif
 #ifndef OPENSSL_NO_EC
     { PROV_NAMES_ECDH, FIPS_DEFAULT_PROPERTIES, ossl_ecdh_keyexch_functions },
-    { PROV_NAMES_X25519, FIPS_DEFAULT_PROPERTIES, ossl_x25519_keyexch_functions },
-    { PROV_NAMES_X448, FIPS_DEFAULT_PROPERTIES, ossl_x448_keyexch_functions },
+    { PROV_NAMES_X25519, FIPS_UNAPPROVED_PROPERTIES, ossl_x25519_keyexch_functions },
```

Comment:
I thought these were approved? - which is why we put them in the fips boundary...

Reply:
Something to raise with the lab I suspect.  The test suite doesn't cover them and the NIST document is still in draft.

@t-j-h thoughts here?

Reply:
@dcooper16 do you have expertise to share?  (David works for NIST computer security division)

Reply:
I've asked the lab what should be done for these.

Reply:
I've heard back, these can stay and have the `FIPS=yes` tag.
I've removed this part from the PR.